packages/core/helper-plugin/lib/src/utils/fetchClient/index.js

```diff
@@ -36,10 +34,14 @@ export const addInterceptors = (instance) => {
 export const fetchClient = ({ baseURL }) => {
   const instance = axios.create({
     baseURL,
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
   });
   addInterceptors(instance);
 
   return instance;
 };
 
-export default fetchClient({ baseURL: process.env.STRAPI_ADMIN_BACKEND_URL });
+export default fetchClient({});
```

Comment:
There is no way to pass `baseURL` anymore. Can we remove the empty object in line 47 and the attributes in line 34?

Reply:
ok

Reply:
done